### ISSUE TITLE
there is no new release

### ISSUE THREAD

**Issue by leventyalcin** [markdownlint/markdownlint#380]
## Description

In order to use markdown_docker hook, I'm consuming the repo from master branch at the moment. However, `pre-commit` is not happy about this and throwing a warning as below;

```shell
[WARNING] The 'rev' field of repo 'https://github.com/markdownlint/markdownlint' appears to be a mutable reference (moving tag / branch).  Mutable references are never updated after first install and are not supported.  See https://pre-commit.com/#using-the-latest-version-for-a-repository for more details.  Hint: `pre-commit autoupdate` often fixes this.
```

I think a regular releases would help.

**Comment by s1e2b3i4** [comment]
Instead, you can pick a particular commit-hash of a commit which includes this change.
For example:

```yaml
  - repo: https://github.com/markdownlint/markdownlint
    rev: 5329a84691ab0fbce873aa69bb5073a6f5f98bdb
    hooks:
      - id: markdownlint_docker
```

Now the warning should be gone.

**Comment by jaymzh** [comment]
You can use tags as well, all our versions are tagged in git. `v0.17.0` for example.
